cairo/kakarot-ssj/crates/evm/src/precompiles/modexp.cairo

```diff
@@ -88,18 +87,158 @@ pub impl ModExp of Precompile {
         let gas = calc_gas(base_len.into(), exp_len.into(), mod_len.into(), exp_highp);
 
         // Padding is needed if the input does not contain all 3 values.
-        let base = input.slice_right_padded(0, base_len);
-        let exponent = input.slice_right_padded(base_len, exp_len);
-
         let (mod_start_idx, _) = base_len.overflowing_add(exp_len);
```

Comment:
note (for us): this should probably not be overflowing_add here. if base_len + exp_len > u32 there's something wrong in the input (which will lead to an OOG prior to that anyway)

Reply:
Should this be added as a comment in code ?

Reply:
what to do?

Reply:
this isn't my code but happy to modify if needed